packages/transcription-devtools/package.json

```diff
@@ -12,9 +12,9 @@
   "type": "module",
   "devDependencies": {},
   "scripts": {
-    "preinstall": "pip install -r requirements.txt",
```

Comment:
While I undertstand `preinstall` script might not be the best place for this, I still think  the `requirements.txt` file as some part to play.